2020/day4/solution.py

```diff
@@ -0,0 +1,57 @@
+import re
+kvp_matcher = re.compile(r"(\S+):(\S+)")
+
+class Passport:
+    RequiredPropertyValidatorMap = {
```

Comment:
```suggestion
    REQUIRED_PROPERTIES = {
```
CamelCase make it read like a nested class.